ecosystem/platform/server/app/components/dialog_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag :dialog, **@rest do %>
   <div class="flex items-center mb-2">
     <h3 class="font-mono uppercase text-2xl"><%= title %></h3>
-    <form method="dialog" class="ml-auto text-right">
+    <form method="dialog" class="ml-auto text-right" autofocus="false">
```

Comment:
It does work only because as you mentioned being boolean it actually set the `autofocus="true" ` to the `<form>`, (which removed focus from the close button). 

I would like to keep appearance consistent across browsers, Safari auto-focusing the close button displaying the blue outline styling when a dialog is opened while Chrome does not. 

Setting autofocus to the first button makes sense, however I'd rather not visually confuse having a focus outline on one button while the other does not. Any objections to setting `autofocus` to the `<div>` container for these buttons or even `<form>` containing the close button?

Reply:
I think the focus outline is fixed in Safari 11.4, where they adopted the heuristics other browsers use to determine whether to show the indicator or not: https://webkit.org/blog/12179/the-focus-indicated-pseudo-class-focus-visible/

> Thanks to tests and discussions, all browsers now follow a common set of heuristics and have the same behavior in most situations. In WebKit, for example, clicking on a `<div tabindex="0">` will no longer show a focus indicator by default (matching other browser engines).

If that's the case, I think we should just leave it as-is since it will be fixed by people updating to the current version of Safari.